tests/test_flows/test_flow_functions.py

```diff
@@ -303,3 +303,21 @@ def test_get_flow_reinstantiate_model_wrong_version(self):
             # ensure that a new flow was created
             assert flow.flow_id is None
             assert "0.19.1" not in flow.dependencies
+
+    def test_get_flow_id(self):
+        clf = sklearn.tree.DecisionTreeClassifier()
+        flow = openml.extensions.get_extension_by_model(clf).model_to_flow(clf).publish()
+
+        self.assertEqual(openml.flows.get_flow_id(model=clf, exact_version=True), flow.flow_id)
+        flow_ids = openml.flows.get_flow_id(model=clf, exact_version=False)
+        self.assertIn(flow.flow_id, flow_ids)
+        self.assertGreater(len(flow_ids), 2)
+
+        flow_ids_exact_version_True = openml.flows.get_flow_id(name=flow.name, exact_version=True)
+        flow_ids_exact_version_False = openml.flows.get_flow_id(
+            name=flow.name,
+            exact_version=False,
+        )
+        self.assertEqual(flow_ids_exact_version_True, flow_ids_exact_version_False)
+        self.assertIn(flow.flow_id, flow_ids_exact_version_True)
+        self.assertGreater(len(flow_ids_exact_version_True), 2)
```

Comment:
Won´t this fail ? Maybe you meant `assertLess`

Reply:
Nope, this works. If only a name is given, exact version does not matter and is ignored.

Reply:
Ups, missed this one.